README.md

```diff
@@ -97,6 +97,34 @@ CONVERSATION_PASSWORD=<add_conversation_password>
 WORKSPACE_ID=<add_conversation_workspace>
 ```
 
+### New API authentication process
+
+The Watson Assistant service has a new API authentication process for service instances that are hosted in the following regions:
+
+* Washington, DC (us-east) as of 14 June 2018
+* Sydney, Australia (au-syd) as of 7 May 2018
+
+For new service instances in the regions listed above, you use IAM for authentication. You can pass either a bearer token or an API key. Tokens support authenticated requests without embedding service credentials in every call. API keys use basic authentication.
+
+![](https://github.com/IBM/pattern-images/raw/master/watson-assistant/watson_assistant_api_key.png)
+
+Replace the existing content of `.env` file to:
+
+```
+# Environment variables
+WORKSPACE_ID=<put Watson Conversation Worksace ID here>
```

Comment:
s/Worksace/Workspace/  I've been emulating the earlier style so this looks OK other than the typo but is this the new preferred template?